doc/rados/operations/control.rst

```diff
@@ -351,17 +355,9 @@ can be useful in certain circumstances: for example, when you are hastily
 balancing in order to remedy ``full`` or ``nearfull`` OSDs, or when there are
 OSDs being evacuated or slowly brought into service.
 
-Operators of deployments that utilize Nautilus or newer (or later revisions of
-Luminous and Mimic) and that have no pre-Luminous clients might likely instead
-want to enable the `balancer`` module for ``ceph-mgr``.
-
-.. note:: The ``balancer`` module does the work for you and achieves a more
-   uniform result, shuffling less data along the way. When enabling the
-   ``balancer`` module, you will want to converge any changed override weights
-   back to 1.00000 so that the balancer can do an optimal job. If your cluster
-   is very full, reverting these override weights before enabling the balancer
-   may cause some OSDs to become full. This means that a phased approach may
-   needed.
+Operators of deployments that utilize Nautilus (or later revisions of Luminous
+and Mimic) and that have no pre-Luminous clients might instead want to enable
+the `balancer`` module for ``ceph-mgr``.
```

Comment:
I think I wrote this several releases ago, and the balancer is on by default now?

Reply:
I will ask later tonight, and return in what I hope is a near-future PR to address this matter.